UnitTests/BraintreePayPalTests/BTPayPalRequest_Tests.swift

```diff
@@ -46,12 +46,6 @@ class BTPayPalRequest_Tests: XCTestCase {
         request.riskCorrelationID = "123-correlation-id"
         request.merchantAccountID = "merchant-account-id"
         request.isShippingAddressEditable = true
-        
-        let lineItem = BTPayPalLineItem(quantity: "1", unitAmount: "1", name: "item", kind: .credit)
```

Comment:
I think we want to keep these  in the unit test to be sure we are passing line items as expected.